doc/pages/faq.asciidoc

```diff
@@ -45,9 +45,30 @@ This variable should be set by the terminal emulator, and not overridden with
 an arbitrary value, otherwise it might interfere with general UI rendering on
 the terminal's window.
 
-Note that if you're using `tmux`, the proper -official- way to get Kakoune to
-work is to add the following line to your `tmux` configuration file:
-`set -sg default-terminal screen-256color`
+== I'm using `tmux` and colors look weird
+
+If you're using a tool that doesn't support "palette switching", colors will
```

Comment:
By "tool", do you mean "terminal emulator"?

Reply:
Well, `tmux` isn't a terminal emulator and yet it has its own terminfo file and has to implement OSC4 itself, so I /think/ I stand correct?

Reply:
To my mind, anything that interprets terminal escape sequences like OSC4 is a terminal emulator; `xterm` is a teminal emulator that draws its output with X11, PuTTY is a terminal emulator that draws its output with Win32, and `tmux` is a terminal emulator that draws its output with ncurses.

Saying "tool" here makes me think of other tools that might or might not support palette switching; Kakoune is a tool that does (in that it will try to set palette entries if it can), Vim (as far as I know) is a tool that does not, but those tools aren't relevant to this section of the docs. Ruling them out with a phrase like "terminal emulator" (or something similar) would make it clearer to me.